pkg/cri/server/container_create_linux_test.go

```diff
@@ -113,11 +114,17 @@ func getCreateContainerTestData() (*runtime.ContainerConfig, *runtime.PodSandbox
 			SecurityContext: &runtime.LinuxSandboxSecurityContext{},
 		},
 	}
-	imageConfig := &imagespec.ImageConfig{
+	imageConfig := imagespec.ImageConfig{
 		Env:        []string{"ik1=iv1", "ik2=iv2", "ik3=iv3=iv3bis", "ik4=iv4=iv4bis=boop"},
 		Entrypoint: []string{"/entrypoint"},
 		Cmd:        []string{"cmd"},
 		WorkingDir: "/workspace",
+		User:       "ContainerUser",
```

Comment:
Do we now need this for linux?

Reply:
No we don't need this for linux since the interpretation of args is different

Reply:
No I meant the User addition of `ContainerUser` that didn't used to be on the spec here